galaxy/postgresql_provision.yml

```diff
@@ -0,0 +1,14 @@
+- hosts: localhost
+  connection: local
+  remote_user: root
+  vars:
+    postgresql_backup_local_dir: /export/postgresql_backup/
+    postgresql_version: 9.3
+    postgresql_conf:
+      listen_addresses: "'*'"
+      hba_file: "'/etc/postgresql/9.3/main/pg_hba.conf'"
+      ident_file: "'/etc/postgresql/9.3/main/pg_ident.conf'"
+    postgresql_pg_hba_conf:
+      - host all all 172.18.0.0/24 md5
```

Comment:
I don't wanted to open it completely to everyone. Would a fixed IP work for you as well? Otherwise let's open it.
On Jul 4, 2016 9:40 AM, Marius van den Beek <notifications@github.com> wrote:In galaxy/postgresql_provision.yml:
> @@ -0,0 +1,14 @@
> +- hosts: localhost
> +  connection: local
> +  remote_user: root
> +  vars:
> +    postgresql_backup_local_dir: /export/postgresql_backup/
> +    postgresql_version: 9.3
> +    postgresql_conf:
> +      listen_addresses: "'*'"
> +      hba_file: "'/etc/postgresql/9.3/main/pg_hba.conf'"
> +      ident_file: "'/etc/postgresql/9.3/main/pg_ident.conf'"
> +    postgresql_pg_hba_conf:
> +      - host all all 172.18.0.0/24 md5

I don't think that's a good idea for people that use swarm, or that manually configure their docker networks (multiple physical hosts for example, that's what we do)... why not set it to 0.0.0.0/0 ?

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.